guvnor-ala/guvnor-ala-openshift/guvnor-ala-openshift-provider/src/test/resources/jboss-image-streams.json

```diff
@@ -0,0 +1,86 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for JBoss Middleware products."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-bpmsuite70-businesscentral-openshift",
+                "annotations": {
+                    "openshift.io/image.insecureRepository": "true",
+                    "openshift.io/display-name": "Red Hat JBoss BPM Suite 7.0 Business Central"
+                }
+            },
+            "spec": {
+                "dockerImageRepository": "ce-os-registry.usersys.redhat.com:5000/qe/bpmsuite70-businesscentral-openshift",
```

Comment:
do we need use this `heavy` image for testing purposes? wondering if a simple hello world wouldn't do the work for `unit tests`